src/services/services.ts

```diff
@@ -6092,7 +6127,7 @@ namespace ts {
                 getReferencesInNode(scope, symbol, declaredName, node, searchMeaning, findInStrings, findInComments, result, symbolToIndex);
             }
             else {
-                const internedName = getInternedName(symbol, node, declarations);
+                const internedName = isConstructor ? declaredName : getInternedName(symbol, node, declarations);

```

Comment:
is the constructor name already interned since it's the parent's symbol? Or does it just not matter if constructor names are interned?